Tests/PackageRegistryTests/RegistryClientTests.swift

```diff
@@ -1497,18 +1405,12 @@ final class RegistryClientTests: XCTestCase {
                 customToolsVersion: .v5_3
             )
             let parsedToolsVersion = try ToolsVersionParser.parse(utf8String: manifest)
-            XCTAssertEqual(parsedToolsVersion, .v5_3)
+            #expect(parsedToolsVersion == .v5_3)
```

Comment:
suggestion (non-blocking): this `do` block, and the one before, are very similar and can be converted to a parameterized tests.

Reply:
Done